dev/reference/tools/data-migration/configure/task-configuration-file-full.md

```diff
@@ -0,0 +1,167 @@
+    - `incremental`: Only replicates the incremental data of the upstream database to the downstream database using the binlog. You can set the `meta` configuration item of the instance configuration to specify the starting position of incremental replication.
+    - `all`: `full` + `incremental`. Makes a full backup of the upstream database, imports the full data to the downstream database, and then uses the binlog to make an incremental replication to the downstream database starting from the exported position during the full backup process (binlog position).
+
+### Feature configuration set
+
+Arguments in each feature configuration set are explained in the comments in the [template](#task-configuration-file-template-advanced).
+
+| Parameter        | Description                                    |
+| :------------ | :--------------------------------------- |
+| `routes` | The routing mapping rule set between the upstream and downstream tables. If the names of the upstream and downstream schemas and tables are the same, this item does not need to be configured. See [Table Routing](/dev/reference/tools/data-migration/features/overview.md#table-routing) for usage scenarios and sample configurations. |
+| `filters` | The binlog event filter rule set of the matched table of the upstream database instance. If binlog filtering is not required, this item does not need to be configured. See [Binlog Event Filter](/dev/reference/tools/data-migration/features/overview.md#binlog-event-filter) for usage scenarios and sample configurations. |
+| `black-white-list` | The filter rule set of the black white list of the matched table of the upstream database instance. It is recommended to specify the schemas and tables that need to be replicated through this item, otherwise all schemas and tables will be replicated. See [Binlog Event Filter](/dev/reference/tools/data-migration/features/overview.md#binlog-event-filter)[Black & White Lists](/dev/reference/tools/data-migration/features/overview.md#black--white-table-lists) for usage scenarios and sample configurations. |
+| `mydumpers` | Configuration arguments of running Mydumper. If the default configuration is sufficient for your needs, this item does not need to be configured. Or you can configure `thread` only using `mydumper-thread`. |
+| `loaders` | Configuration arguments of running Loader. If the default configuration is sufficient for your needs, this item does not need to be configured. Or you can configure `pool-size` only using `loader-thread`. |
+| `syncers` | Configuration arguments of running Syncer. If the default configuration is sufficient for your needs, this item does not need to be configured. Or you can configure `worker-count` only using `syncer-thread`. |
+
+## Instance configuration
+
+This part defines the subtask of data replication. DM supports replicating data from one or multiple MySQL instances to the same instance.
```

Comment:
to the same instance -> **in the upstream** to the same instance **in the downstream.**